src/content/pages/explore.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: More Tips on Exploring Prague
+subtitle: In EuroPython there are many attendees will come with their family. While not attending the conference, why not go and explore Prague and the surrounding area?
```

Comment:
I'd make paste everything into ChatGPT and ask it to fix the grammar only - I see lots of typos, grammar mistakes etc.